kolibri/core/assets/src/core-app/constructor.js

```diff
@@ -25,32 +32,18 @@ const publicMethods = [
 ];
 
 /**
- * Constructor for lib object that exposes libraries that are shared across all plugins.
- * In addition to being added as properties of kolibriGlobal.lib, they are also made
- * available to be require'd in other webpack-loaded modules and apps. This behavior
- * is configured in webpack.config.js
- * @classdesc
- */
-function Lib() {
-  // libraries
-  this.logging = require('../logging');
-  this.vue = vue;
-  this.vuex = vuex;
-  this.conditionalPromise = require('../conditionalPromise');
-}
-
-/**
  * Constructor for object that forms the public API for the Kolibri
  * core app.
  * @constructor
  */
 module.exports = function CoreApp() {
-  this.lib = new Lib();
+  Object.assign(this, constructorExport());
+
   this.resources = new ResourceManager(this);
   const mediator = new Mediator();
 
-  this.constants = require('../constants');
-  this.coreActions = require('../core-actions');
+  this.client = rest.wrap(mime, { mime: 'application/json' }).wrap(csrf, { name: 'X-CSRFToken',
+      token: cookiejs.get('csrftoken') }).wrap(errorCode);
```

Comment:
(I guess it could be attached with a getter, so it would be created on-the-fly every time it's accessed?)